_pages/how-we-work/equipment.md

```diff
@@ -62,6 +62,10 @@ TTS employees are issued an Apple iPhone.  The TTS Talent Team will already have
 - To reduce waste, GSA IT will [disable mobile service for inactive accounts](https://insite.gsa.gov/portal/content/663074). The consumed minutes is the metric used to determine activity, so if you only use the phone for data, you'll still need to make an occasional call on your mobile phone to prevent the account from going inactive.
 - If you need help with your phone, call the GSA IT Help Desk at 1-866-450-5250.
 
+## Software
+
+See the [software request guide](https://docs.google.com/document/d/1AUyRggdKWLRcHC3E4ssHE_2tmOaDYTv0ex0lgwbwFSA/edit#).
```

Comment:
The TTS-specific parts of that doc will move here, or to a dedicated page if long enough.